examples/api/lib/material/progress_indicator/circular_progress_indicator.0.dart

```diff
@@ -52,19 +51,32 @@ class _ProgressIndicatorExampleState extends State<ProgressIndicatorExample> wit
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Padding(
-        padding: const EdgeInsets.all(20.0),
+      body: Center(
         child: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          spacing: 16.0,
+          mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
-            Text(
-              'Circular progress indicator with a fixed color',
-              style: Theme.of(context).textTheme.titleLarge,
+            const Text('Determinate CircularProgressIndicator'),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16),
+              child: CircularProgressIndicator(
+                year2023: year2023,
+                value: controller.value,
+              ),
             ),
-            CircularProgressIndicator(
-              value: controller.value,
-              semanticsLabel: 'Circular progress indicator',
+            const Text('Indeterminate CircularProgressIndicator'),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16),
+              child: CircularProgressIndicator(year2023: year2023),
             ),
+            SwitchListTile(
+              value: year2023,
+              title: const Text('Toggle year2023 flag'),
```

Comment:
nit: Maybe we can provide some more info by this switch:) like
`year2023 ? const Text('Switch to latest M3 style') : const Text('Switch to year2023 M3 style');`